optimize.c

```diff
@@ -2057,7 +2057,7 @@ convert_code_r(compiler_state_t *cstate, conv_state_t *conv_state,
 		}
 	}
 	src = p->stmts;
-	for (off = 0; off < slen && src; off++) {
+	for (off = 0; off < (u_int) slen && src; off++) {
```

Comment:
In 1a796886890fb989e2071d63d9d0279a046ec62c I made `slen` unsigned, so the cast is no longer necessary.